site/docs/guides/contributor/release-process/egeria-core-release.md

```diff
@@ -0,0 +1,58 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the Egeria project 2020. -->
+
+### Egeria Core Release process
+
+#### Obtaining releases / artifacts for Core Egeria
+| Location | Usage |
+|---|---|
+| [Maven Central core Egeria release :material-dock-window:](https://mvnrepository.com/artifact/org.odpi.egeria){ target=maven } | typically used by other developers integrating with our code |
+| [Github core Egeria Release :material-github:](https://github.com/odpi/egeria/releases){ target=gh } | source code in `zip` and `tar.gz` formats |
+| `git` | `git checkout Vx.y` to get version as-shipped (each release is tagged at the point it is shipped) |
+
+### Core Egeria Release process
+
```

Comment:
Open up an issue in core egeria using the 'release' template. This contains check boxes for the activities that should be performed

(Note: you may want to review this - see .github/ISSUE_TEMPLATES)